internal/conf/configuration.go

```diff
@@ -452,11 +452,25 @@ type HookConfiguration struct {
 	CustomSMSProvider           ExtensibilityPointConfiguration `json:"custom_sms_provider" split_words:"true"`
 }
 
+type HTTPHookSecrets []string
+
+func (h *HTTPHookSecrets) Decode(value string) error {
+	parts := strings.Split(value, ":")
+	for _, part := range parts {
+		if part != "" {
+			*h = append(*h, part)
+		}
+	}
+
+	return nil
+}
+
 type ExtensibilityPointConfiguration struct {
-	URI             string   `json:"uri"`
-	Enabled         bool     `json:"enabled"`
-	HookName        string   `json:"hook_name"`
-	HTTPHookSecrets []string `json:"secrets"`
+	URI      string `json:"uri"`
+	Enabled  bool   `json:"enabled"`
+	HookName string `json:"hook_name"`
+	// We use : as a separator for keys and ; as a separator for keys within a keypair. For instance: v1,whsec_test;v1a,whpk_myother;v1a,whsk_testkey
```

Comment:
i feel like it would be very challenging for us to differentiate between `:` and `;` if there are typo-issues - the example also doesn't include one with `:`

Reply:
Sure, that's a fair point. What about `|` ?

Reply:
Or maybe the other way round might be better: with `|` as a seperator for keys and `:` as a joiner for keypairs

whsec_test|v1a,whpk_myother:v1a,whsk_testkey

Reply:
yup lgtm!

Reply:
Please update the PR description.